packages/react-composites/src/composites/CallComposite/components/MediaGallery.tsx

```diff
@@ -192,6 +195,7 @@ export const MediaGallery = (props: MediaGalleryProps): JSX.Element => {
     return (
       <VideoGallery
         {...videoGalleryProps}
+        videoTilesOptions={videoTilesOptions}
```

Comment:
nit: Don't forget to add this to the deps array

Reply:
Yup! Did this just now after seeing the linter issues